packages/ui/src/core/check-box/check-box.props.d.ts

```diff
@@ -1,9 +1,12 @@
 import { ChangeEvent } from 'react';
+import { CheckBoxSizeType } from '../check-box-group/check-box-group.props';
 
 export interface CheckBoxProps extends React.HTMLAttributes<HTMLSpanElement> {
   checked?: boolean;
   onChange?: (event: ChangeEvent<HTMLInputElement>) => void;
   checkBoxProps?: React.InputHTMLAttributes;
   readOnly?: boolean;
   disabled?: boolean;
+  isHovering?: boolean;
```

Comment:
We shouldn't add internal props to prop files. Because it will show to any one who wants to use it